.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
   build:
     <<: *node_executor
-    resource_class: medium+
+    resource_class: large
```

Comment:
Could you please explain the findings behind this change?

Reply:
Had to increase the size as CI build kept failing due to CPU being maxxed out. Will need to optimize this build as well and then we can lower the class to medium+ again.